### DISCUSSION TITLE
KEEPER - 로그아웃 불가 및 게시글 확인 불가 문제 해결, 회고 - 2024.01.29 ~ 2024.02.04

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#331] [5. 회고]
https://02ggang9.github.io/keeper/logoutProblem/

https://02ggang9.github.io/review/review10/

**Comment by shkisme** [comment]
짱구 짱. 👍

드뎌 리드미를 썼따~~~~ 해피 ^______^ 수고많았어!! 그저 GOAT 짱구.

**Reply by 02ggang9** [reply]
ReadME 분량 많은데 누나도 수고 많았어~ 👍👍 GOAT 보름 ㄷ.ㄷ
